client/src/components/Header.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import Logo from './Logo';
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: left;
```

Comment:
```suggestion
  justify-content: space-between;
```

Currently, if you view your app on the iPad viewport in storybook, you see that your header is all on the left of the screen although your width is `100%`. With `space-between` you are able to stretch it to the whole screen and will adapt to different screen sizes.

Reply:
no idea why I chose `justify-content: left` 😅  I tried it out and I actually like the `justify-content: center` option more since then everything is nice together but still aligned 👍 thank you for the heads up though!